tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -338,3 +339,53 @@ def test_fetch_stats_for_all_services_by_date_range(notify_db_session):
     assert not results[4].notification_type
     assert not results[4].status
     assert not results[4].count
+
+
+@freeze_time('2018-01-04 14:00')
+def test_fetch_monthly_template_usage_for_service(sample_service):
+    template_one = create_template(service=sample_service, template_type='sms', template_name='1_one')
+    template_two = create_template(service=sample_service, template_type='email', template_name='2_two')
+    template_three = create_template(service=sample_service, template_type='letter', template_name='3_three')
+
+    create_ft_notification_status(bst_date=date(2018, 1, 1),
+                                  service=sample_service,
+                                  template=template_one,
+                                  count=2)
+    create_ft_notification_status(bst_date=date(2018, 2, 1),
+                                  service=sample_service,
+                                  template=template_two,
+                                  count=4)
+    create_ft_notification_status(bst_date=date(2018, 3, 1),
+                                  service=sample_service,
+                                  template=template_three,
+                                  count=5)
+    create_notification(template=template_one)
```

Comment:
can we add a notification from yesterday for one of the other templates so that we can make sure that one isn't picked up?